summary_workflows/quantification/specification/example_files/assessment_out.json

```diff
@@ -1,46 +1,46 @@
 [
```

Comment:
I'm afraid this example file is not correct anymore if you loop over `windows` AND `return_df_types` in `compute_metrics.py`. However, it would be quite an overhead to specify all metrics combinations. So maybe you can add some "..." and specify at least each return type once. Or instead of writing the exact id/name, write it with [METRIC_NAME] and [RETURN_TYPE] or sth like that.

Reply:
I cannot add comments in json files. So I can add all metrics that are also in the example, in total 27.